server.go

```diff
@@ -5300,6 +5316,33 @@ func (s *server) SendCustomMessage(peerPub [33]byte, msgType lnwire.MessageType,
 	return peer.SendMessageLazy(true, msg)
 }
 
+// SendOnionMessage sends a custom message to the peer with the specified
+// pubkey.
+// TODO(gijs): change this message to include path finding.
+func (s *server) SendOnionMessage(peerPub [33]byte,
```

Comment:
wonder if we should not have a dedicated OnionMessageServer? instead of adding this logic directly to the main `server` struct.

Reply:
you mean having a separate sub-rpc-server for onion-messages ?